handbook/working_and_growing_here.md

```diff
@@ -226,13 +248,122 @@ After you go through the interview process, you will hear back from the hiring m
+
+#### How we want to do this differently
+
+This problem has been something that I’ve been hyper-focused on since starting Fishtown Analytics. I saw the negative impacts of this previously and wanted to steer hard in the opposite direction. Connor and Drew and I have focused on building a culture that emphasizes cross-departmental collaboration since the very beginning, and when we’ve hired or promoted people into leadership roles we’ve heavily focused on people who are wired to be collaborative.
+
+As a result, we’ve had notable successes on this front in our short life as a company: the dbt community and dbt Learn are both programs that live in a highly cross-functional space. It’s a tremendous credit to everyone involved in these programs for operating in a zone of some level of ambiguity.
+
+But even with this success, it’s naive to assume that just “hiring collaborative people” is an answer to a systems problem. Ultimately, we’re all products of the systems we inhabit, and we have to design those systems well in order to get the outcomes we want. Here are my suggestions to continue to keep Fishtown Analytics focused on cross-departmental collaboration over the long run:
+
+- **Establish clear cross-departmental budgetary mechanisms.** For example, make a clear mechanism whereby marketing can allocate budget for a data analyst hire that will ultimately live on the data team. Note that this needs to handle the scenario where multiple departments want to contribute a % of an FTE towards a single full-time hire.
+- **Minimize executive compensation that is tied to anything but overall equity value.** We don’t have to remove incentive-based compensation that is tied to specific metrics altogether, but it should always be a distant personal priority for every executive when weighed against the success of the overall business.
+- **Create a new company value: “We share our legos.”** Culture is one of the biggest tools we have to shape behavior, and our values are the best tool we have for shaping culture.
+- **Answer the hard questions about cross-departmental roles up-front.** More on this in Appendix A.
+- **Abolish fiefdoms.** Kill the idea that departmental strategy is set and budgets are allocated at the sole discretion of the leader of that department. All leaders can weigh in on the strategic choices and budgets of other departments, and these concerns need to be addressed prior to moving ahead with the proposed plans. Use the OKR-setting process to make sure that goals are aligned across different departments.
+
+#### Next steps
```

Comment:
This section should be removed

Reply:
Do you want appendix A also removed? (Line 445)